src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    public Task(String description,boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+    public void completeTask(){
+        isDone = true;
+    }
+    @Override
+    public String toString(){
+        return"["+this.getStatusIcon()+"] "+this.description;
```

Comment:
To enhance readability, perhaps you can consider putting a space before and after the `+`.